test/unit/keystore/eth2.spec.ts

```diff
@@ -31,53 +30,57 @@ describe("Eth1ICGKeystore", () => {
         sandbox.restore();
     });
 
+
     it("should create keystore", async () => {
-        const priv = PrivateKey.fromHexString(privateKeyStr);
+        const priv = PrivateKey.fromHexString(privateKey);
         const keypair = new Keypair(priv);
 
         const writeStub = sandbox.stub(fs, "writeFileSync");
         const readStub = sandbox
             .stub(fs, "readFileSync")
             .withArgs(keyStoreFilePath)
             .returns(
-                JSON.stringify(await example)
+                await JSON.stringify(example)
             );
 
-        eth1Keystore = await Eth1Keystore.create(keyStoreFilePath, password, keypair);
+        eth2Keystore = await Eth2Keystore.create(keyStoreFilePath, password, keypair);
+        eth2Keystore.decrypt(password);
         expect(writeStub.calledOnce).toEqual(true);
         expect(readStub.calledOnce).toEqual(true);
+
     });
 
+
     it("should decrypt", async () => {
-        const keypair = await eth1Keystore.decrypt(password);
+        const keypair = await eth2Keystore.decrypt(password);
```

Comment:
That slipped me with first version of keystore. Could you make test independent of each other using beforeEach or something